content/docs/user-guide/networking/multiplayer/split.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Client and Server Launchers with the Multiplayer Gem
+description: Build Open 3D Engine launchers with the Multiplayer Gem specifically targeting Clients, Servers or both.
+linkTitle: Separating Client and Server
+---
+
+The Multiplayer Gem supports builds that contain only client logic, only server logic, or both client and server logic. This allows users to create executables of smaller size by excluding unnecessary logic and dependencies. It also allows hiding potentially sensitive logic unique to one executable from the other. 
+
+* GameLauncher is a client only launcher
+* ServerLauncher is a server only launcher suitable for Dedicated Servers
+* UnifiedLauncher provides both and is suitable for Client Hosted Servers
```

Comment:
It it possible to turn off/not generate this target? ie if you never need a unified launcher (which I am assuming is the majority use case outside of testing)

Reply:
The target will always be generated as part of the VS solution but there is no requirement to build it. A user can opt not to build the UnifiedLauncher if they want.